tests/behat/features/plugin-check-severity.feature

```diff
@@ -192,5 +192,19 @@ Feature: Test that the severity level in plugin check works.
       upgrade_notice_limit,WARNING,5
       """
 
+    When I run the WP-CLI command `plugin check foo-bar-wp --format=csv --fields=code,type,severity --error-severity=7 --include-error-severity`
```

Comment:
May be I am missing something but I don't see this scenario testing the converted ERROR to WARNING. Which ERROR with less than 7 severity is converted to WARNING here?

Reply:
yes, it's on the results.